diff --git a/node_modules/react-dev-utils/WebpackDevServerUtils.js b/node_modules/react-dev-utils/WebpackDevServerUtils.js
index 6a56f66..4ae7b86 100644
--- a/node_modules/react-dev-utils/WebpackDevServerUtils.js
+++ b/node_modules/react-dev-utils/WebpackDevServerUtils.js
@@ -225,7 +225,7 @@ function createCompiler({
     const messages = formatWebpackMessages(statsData);
     const isSuccessful = !messages.errors.length && !messages.warnings.length;
     if (isSuccessful) {
-      console.log(chalk.green('Compiled successfully!'));
+      console.log(chalk.green(`Compiled successfully in ${stats.endTime - stats.startTime}ms!`));
     }
     if (isSuccessful && (isInteractive || isFirstCompile)) {
       printInstructions(appName, urls, useYarn);
@@ -246,7 +246,7 @@ function createCompiler({
 
     // Show warnings if no errors were found.
     if (messages.warnings.length) {
-      console.log(chalk.yellow('Compiled with warnings.\n'));
+      console.log(chalk.yellow(`Compiled in ${stats.endTime - stats.startTime}ms with warnings.\n`));
       console.log(messages.warnings.join('\n\n'));
 
       // Teach some ESLint tricks.
